examples/2d/elixir_euler_unstructured_quad_basic.jl

```diff
@@ -12,11 +12,11 @@ initial_condition = initial_condition_convergence_test
 source_terms = source_terms_convergence_test
 
 boundary_condition_convergence_test = BoundaryConditionDirichlet(initial_condition)
-boundary_conditions = Dict( "Slant" => boundary_condition_convergence_test,
-                            "Bezier" => boundary_condition_convergence_test,
-                            "Right"  => boundary_condition_convergence_test,
-                            "Bottom" => boundary_condition_convergence_test,
-                            "Top"    => boundary_condition_convergence_test )
+boundary_conditions = Dict( :Slant  => boundary_condition_convergence_test,
+                            :Bezier => boundary_condition_convergence_test,
+                            :Right  => boundary_condition_convergence_test,
+                            :Bottom => boundary_condition_convergence_test,
+                            :Top    => boundary_condition_convergence_test )
```

Comment:
Does this switch to symbols really bring any improvements in terms of performance or allocations? AFAICT from the benchmarks, they didn't really improve over this change. I am asking, since intuitively I would also think that symbols might be faster, but Julia has it's own performance rules, as I have found out many, many times...

Also, it would be helpful if @andrewwinters5000 could weigh in here: Are there BC names supported by HOHQMesh that are not representable as a symbol (at least not easily)? E.g. "left wall" or similar

Reply:
As discussed on Zoom, this is not an issue since boundaries should always be a single word

Reply:
Will still work...
```julia
julia> Symbol("left wall"^100)
Symbol("left wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wallleft wall")
```